.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - nightly
```

Comment:
As PHP 8 has just been released this probably can be changed to 8.0

Reply:
yep, true. just added "8.0" 👍